README.md

```diff
@@ -24,10 +24,6 @@ Please note that we do not provide support for any of the unofficial libraries,
 
 [go-yelp](https://github.com/JustinBeckwith/go-yelp)
 
-#### Node.js
-
-[node-yelp](https://github.com/Yelp/yelp-api/tree/master/v2/node.js)
```

Comment:
Good catch!  Fallout from #67 .

I think we actually should link to https://github.com/olalonde/node-yelp rather than remove the link, though.